lessons/puzzles/One.py

```diff
@@ -0,0 +1 @@
+print('hello world')
```

Comment:
`nit`, please add a newline to the file

```suggestion
print('hello world')

```